tests/base/symmetric_tensor_39.cc

```diff
@@ -44,6 +44,9 @@ int main()
           counter += 1.0;
         }
 
+  // without this we get wrong results with clang in release mode
+  std::cout << '\0';
```

Comment:
it's in the terminal output but NOT in the log log, which is used here via `deallog`.